README.md

```diff
@@ -1,4 +1,5 @@
-# &lt;gaia-toolbar&gt;
+# &lt;gaia-toolbar&gt; [![](https://travis-ci.org/gaia-components/gaia-toolbar.svg)](https://travis-ci.org/gaia-components/gaia-toolbar) [![devDependency Status](https://david-dm.org/gaia-components/gaia-header/dev-status.svg)](https://david-dm.org/gaia-components/gaia-toolbar#info=devDependencies)
```

Comment:
Please can we drop the devDependencies label. I don't think it communicates anything of any value.